### DISCUSSION TITLE
RangeNode exclude_end

### DISCUSSION THREAD

**Discussion by FnControlOption** [ruby/prism#996] [General]
Currently, the only way to tell the difference between an inclusive range `a..b` and an exclusive range `a...b` is to check whether `operator_loc.end - operator_loc.start` is either two or three. Would it be worthwhile to add an `exclude_end` boolean/flag to `RangeNode`?

**Comment by kddnewton** [comment]
Yes, that would be a good place for flags for sure. You can copy what we recently added to call nodes for safe navigation.

**Comment by eregon** [comment]
That's fixed now.
